util/error.go

```diff
@@ -7,3 +7,7 @@ import "fmt"
 func InvalidCommand(f string) error {
 	return fmt.Errorf("invalid command: Flag '--%s' is not set or is empty", f)
 }
+
+func InvalidFlag(f string) error {
+	return fmt.Errorf("invalid command: Flag '--%s' is not valid", f)
```

Comment:
i think it would be nice to call out that the value that was provided is not valid. otherwise it could be mistaken for `--type` itself not being valid.